pytrip/utils/cubeslice.py

```diff
@@ -90,17 +88,17 @@ def load_ct_cube(filename):
     """
 
     if not filename:
-        logger.warn("Empty CT cube filename")
+        logger.warning("Empty CT cube filename")
         return None, None
 
     logger.info("Reading " + filename)
-    ctx_header, _ = pt.CtxCube.parse_path(filename)
+    ctx_header = pt.CtxCube.header_file_name(filename)
 
     if ctx_header is None:
-        logger.warn("Path " + filename + " doesn't seem to point to proper CT cube")
+        logger.warning("Path " + filename + " doesn't seem to point to proper CT cube")
```

Comment:
Not really - the same warning will pop-up if you point to `mp3` file instead of `.hed`